src/App.tsx

```diff
@@ -129,6 +129,9 @@ function App() {
 
   const loadApp = useCallback(
     loadProvider => {
+      if (networkId == -1) {
```

Comment:
Do you know where that -1 is coming from? 

I was debugging that same issue last night, seems that was the reason to have that `API keys must be set in the .env` error since getSelfHostedNode for some reason would fire twice and second run would have that -1 there and return an empty URI_LIST.